webpack.dev.config.js

```diff
@@ -1,7 +1,16 @@
+const keycloakHost = '<INSERT_YOUR_IP>';
+const keycloakUser = 'astark';
+const keycloakPassword = 'astark';
+const keycloakClientId = 'shogun-app';
```

Comment:
Can we make use of environment variables? so that we do not accidently commit credentials in the future?

maybe sth like (pseudocode):

```js
const keycloakPassword = fromEnvironment('KEYCLOAK_PASS');
```

And in `fromEnvironment` we warn loudly when there is no such environment variable and fail to start at all?